youtube_dl/extractor/platzi.py

```diff
@@ -186,10 +169,12 @@ def _real_extract(self, url):
 
         webpage = self._download_webpage(url, course_name)
 
+        initialData = self._search_regex(
```

Comment:
This would fail more gracefully:
```
        props = self._parse_json(initialData, course_name, default={})
        props = try_get(props, lambda x: x['initialProps'], dict) or {}
        entries = []
        for chapter_num, chapter in enumerate(props.get('concepts') or [], 1):
```
Or if you want to flag the absence of concepts expand the last line
```
        concepts = try_get(props, lambda x: x['concepts'], list)
        if concepts is None:
            raise ExtractorError('No concepts found', expected=True)
        for chapter_num, chapter in enumerate(concepts, 1):
```